packages/ra-ui-materialui/src/list/filter/FilterButton.spec.tsx

```diff
@@ -92,7 +92,7 @@ describe('<FilterButton />', () => {
             );
         });
 
-        it('should remove all filters when the "Clear all filters" button is clicked', async () => {
+        it('should remove all non alwaysOn filters when the "Remove all filters" button is clicked', async () => {
```

Comment:
I would not have changed the title of this test.

Rather, I would have had:
- Test 1: `should remove all filters when the "Remove all filters" button is clicked`
- Test 2 (the one you added): `should also remove all alwaysOn filters when the "Remove all filters" button is clicked`

But you are right that `Clear all filters` was renamed to `Remove all filters` so we should keep this part